packages/frontend/src/ee/features/aiCopilot/components/ChatElements/AgentChatBubbles.tsx

```diff
@@ -184,54 +211,52 @@ export const AssistantBubble: FC<{
-                            color="gray"
-                            aria-label="upvote"
-                            onClick={handleUpvote}
-                            display={isLoading ? 'none' : 'block'}
-                        >
-                            <MantineIcon
-                                icon={upVoted ? IconThumbUpFilled : IconThumbUp}
-                            />
-                        </ActionIcon>
-                    )}
+                {(!hasRating || upVoted) && (
+                    <ActionIcon
+                        variant="subtle"
+                        color="gray"
+                        aria-label="upvote"
+                        onClick={handleUpvote}
+                        display={isLoading ? 'none' : 'block'}
+                    >
+                        <MantineIcon
+                            icon={upVoted ? IconThumbUpFilled : IconThumbUp}
+                        />
+                    </ActionIcon>
+                )}
 
-                    {(!hasRating || downVoted) && (
-                        <ActionIcon
-                            variant={'transparent'}
-                            color="gray"
-                            aria-label="downvote"
-                            onClick={handleDownvote}
-                            display={isLoading ? 'none' : 'block'}
-                        >
-                            <MantineIcon
-                                icon={
-                                    downVoted
-                                        ? IconThumbDownFilled
-                                        : IconThumbDown
-                                }
-                            />
-                        </ActionIcon>
-                    )}
-                </Group>
+                {(!hasRating || downVoted) && (
+                    <ActionIcon
+                        variant="subtle"
+                        color="gray"
+                        aria-label="downvote"
+                        onClick={handleDownvote}
+                        display={isLoading ? 'none' : 'block'}
+                    >
+                        <MantineIcon
+                            icon={
+                                downVoted ? IconThumbDownFilled : IconThumbDown
+                            }
+                        />
+                    </ActionIcon>
+                )}
+                {!!vizQuery.data?.openInExploreUrl && (
+                    <Button
+                        variant="subtle"
+                        color="gray"
+                        size="xs"
+                        aria-label="open in explore"
+                        leftSection={<MantineIcon icon={IconExternalLink} />}
+                        component={Link}
+                        to={vizQuery.data.openInExploreUrl}
+                        target="_blank"
+                        style={{
+                            color: '#868e96',
```

Comment:
nice if we had this in mantine theme with a name/token

Reply:
yes! I need to investigate why Buttons change colors when displaying as Links 🤔